jca/WFLY-17180_Add_queryTimeout_validation_parameter.adoc

```diff
@@ -0,0 +1,72 @@
+= WFLY-17180 Add query-timeout validation parameter
```

Comment:
Please add appropriate Jekyll front matter so this ends up correctly categorized on the docs.wildfly.org/wildfly-proposals page.